platform/api/src/api/v1/request_context.rs

```diff
@@ -1,35 +1,39 @@
 use std::{collections::HashMap, sync::Arc};
 
-use crate::database::{role, session, user};
 use tokio::sync::RwLock;
-use uuid::Uuid;
+use ulid::Ulid;
 
-use crate::{api::v1::gql::error::Result, global::GlobalState};
+use crate::{
+    api::v1::gql::error::Result,
+    database::{Role, RolePermission, Session, User},
+    global::GlobalState,
+};
 
 #[derive(Clone)]
 pub struct AuthData {
-    pub session: session::Model,
-    pub user_roles: Vec<role::Model>,
-    pub user_permissions: role::Permission,
+    pub user: User,
```

Comment:
Why did you add user here?

Reply:
Because we had it as a byproduct. I wasnt sure if we needed it

Reply:
I excluded it on purpose because it could have outdated information after a few requests

Reply:
Ah, okay.